packages/actions/test/unit/authentication.test.ts

```diff
@@ -44,8 +44,12 @@ describe("Authentication", () => {
     describe("getCurrentFirebaseAuthUser()", () => {
```

Comment:
The first test from 00-auth was not moved to this file. Any reason why?

Reply:
I thought [this test](https://github.com/quadratic-funding/mpc-phase2-suite/blob/0e2794492b3e0298f755b529eed61756fb779a4e/packages/actions/test/unit/authentication.test.ts#L76-L91) does have same semantic, that desired user info has registered correctly and could be retrieved, with the test what you are pointing that missed.

Reply:
Fair enough, that makes sense.